commit 97e0ae8191501c72970904cdaf9ebb9e92627e3d (refs/changes/75/4575/1)
Author: Rui Loura <rui@joyent.com>
Date:   2018-07-25T18:53:28+00:00 (1 year, 2 months ago)
    
    TRITON-587 net-agent needs updated regex for rack tagged admin nics (bump version)

diff --git a/package.json b/package.json
index c16fdea..e29dd92 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "net-agent",
     "description": "Triton Network Agent",
-    "version": "2.0.0",
+    "version": "2.1.0",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
